test/functional/ui/float_spec.lua

```diff
@@ -1105,6 +1107,191 @@ describe('floating windows', function()
       end
     end)
 
+    it('does not crash with inccommand #9379', function()
+      screen:try_resize(30,15)
```

Comment:
Is there any particular reason for this? It would be more readable if the test would use the default of 40 columns and 7 lines.

Reply:
I believe I got `30,15` from one of the tests in `inccommand_spec`.

Updated to use `40,7`.